panels/search/cc-search-panel.c

```diff
@@ -657,10 +696,27 @@ search_providers_discover_thread (GTask *task,
                                   gpointer task_data,
                                   GCancellable *cancellable)
 {
+  DiscoverProvidersResult *result = NULL;
   GList *providers = NULL;
   const gchar * const *system_data_dirs;
   int idx;
+  EpcAppFilter *filter = NULL;
```

Comment:
Is it not possible to use `g_autoptr(EpcAppFilter)`?

Reply:
None of the other code in `cc-search-panel.c` uses it, so I was being consistent. But I can use `g_autoptr()` there if you would prefer.